stdlib/private/StdlibUnittest/LoggingWrappers.swift.gyb

```diff
@@ -439,8 +439,7 @@ public struct Logging${Kind}<Base: ${Kind}Type> : ${Kind}Type, LoggingType {
     @noescape body: (UnsafeMutablePointer<Generator.Element>, Int) throws -> R
   ) rethrows -> R? {
     ++Log._withUnsafeMutableBufferPointerIfSupported[selfType]
-    let result = try base._withUnsafeMutableBufferPointerIfSupported(body)
-    if result != nil {
+    if let result = try base._withUnsafeMutableBufferPointerIfSupported(body) {
       ++Log._withUnsafeMutableBufferPointerIfSupportedNonNilReturns[selfType]
     }
     return result
```

Comment:
Did you compile the code and run tests?  I don't think this will compile, the `result` variable is not in scope anymore.